frontend/src/pages/profile/components/PanelDataUpload.jsx

```diff
@@ -401,6 +409,38 @@ const PanelDataUpload = () => {
               }}
             />
           </TabPane>
+          <TabPane tab="Approved" key={"approved-batch"}>
```

Comment:
Instead of repeating the component, we can create new component Table based on the pending batch and reuse it